CMakeLists.txt

```diff
@@ -103,6 +96,27 @@ elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
     string(APPEND CMAKE_CXX_FLAGS " -fmax-errors=3")
 endif()
 
+if (USE_CONAN)
+    message("Using conan package manager")
+    if(NOT EXISTS "${CMAKE_BINARY_DIR}/conan.cmake")
+        message(STATUS "Downloading conan.cmake from https://github.com/conan-io/cmake-conan")
+        file(DOWNLOAD "https://raw.githubusercontent.com/conan-io/cmake-conan/master/conan.cmake"
```

Comment:
I'm not totally opposed to this, but I think it may be better to copy the cmake file and ship it with our code, as we do with other cmake files. The rationale is twofold:
 1. We won't get updates if the file already exists
 2. An update of the remote file can break our build if the user doesn't already have it.

Reply:
done

Reply:
Since we no longer need to download, and conan.cmake will always exist, can you please remove this conditional?